packages/driver-test-support/lib/e2e-suite.js

```diff
@@ -99,8 +99,7 @@ export function createSessionHelpers(port, address = TEST_HOST) {
 
 /**
  * Creates E2E test suites for a driver.
- * @template {Driver} P
- * @param {DriverClass<P>} DriverClass
+ * @param {DriverClass} DriverClass
```

Comment:
I loosened this b/c we don't really care about the specifics.  When consuming `Driver`, `ExternalDriver`, `DriverClass`, etc., the question to ask is "do I care what the capabilities are?" If the answer is "yes", use a type param.  If the answer is "no", don't.